Lib/fontTools/feaLib/parser.py

```diff
@@ -10,13 +10,15 @@
 import re
 
 
+
```

Comment:
sorry to be such a pain.. but maybe you could set your text editor to not add/remove empty lines at random?